models/v1beta1/model/model.go

```diff
@@ -100,6 +100,14 @@ type ModelDefinition struct {
 	// To prevent cyclic error, the type is changed to interface, it doesn't affect registration
```

Comment:
I wonder why the model.go file is under a v1beta1 directory. This is unnecessary and will become antiquated and confusing as the schema progresses in version number.